postgresql-standalone-server-ubuntu/README.md

```diff
@@ -0,0 +1,51 @@
+# PostgreSQL Server on Ubuntu VM
+
+<a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2Fpostgresql-standalone-server-ubuntu%2Fazuredeploy.json" target="_blank"><img src="http://azuredeploy.net/deploybutton.png"/></a>
+
+This template uses the Azure Linux CustomScript extension to deploy a postgresql server. It creates an Ubuntu VM, does a silent install of postgresql server, version:9.3.5, and the basic configuration is below:
+
+PGPORT=1999
```

Comment:
sorry, don't have code blocks. hence to use description instead of showing the variables.